ui/css/oae.signup.css

```diff
@@ -0,0 +1,310 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+main > h3 {
+    border-bottom: 1px #EEEEEE solid;
```

Comment:
The border is too close to the actual title. It should look the same as the "Recent activity" title in the me space.

<img width="1395" alt="screen shot 2015-09-10 at 16 14 24" src="https://cloud.githubusercontent.com/assets/109850/9803399/068e058c-57d7-11e5-8e20-dc4345c26ec0.png">